e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -172,15 +172,18 @@ var _ = Describe("App With Domain", func() {
 			// Make sure the response is OK.
 			var resp *http.Response
 			var fetchErr error
-			Eventually(func() (int, error) {
-				resp, fetchErr = http.Get(route.URL)
-				return resp.StatusCode, fetchErr
-			}, "60s", "1s").Should(Equal(200))
-			// HTTP should route to HTTPS.
-			Eventually(func() (int, error) {
-				resp, fetchErr = http.Get(strings.Replace(route.URL, "https", "http", 1))
-				return resp.StatusCode, fetchErr
-			}, "60s", "1s").Should(Equal(200))
+			urls := strings.Split(route.URL, " or ")
```

Comment:
:gorilla-think: Does "and" better describe it than "or". Both of the aliases need to work right?

Reply:
Yeah the reason why i use `or` instead of `and` is because `and` means you need to use both of them to make it work while `or` usually means you can pick one from the two. Not a big deal tho I think either using `and` or `or` they should understand what it means...